lib/model/card/quest/into_the_depths_4_utopia.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:treasure_of_the_high_seas/model/game_result.dart';
+
+import '../../resource.dart';
+import '../action/card_action.dart';
+import '../action/end_game_action.dart';
+import '../card.dart';
+
+class Utopia extends Card {
+  const Utopia() : super("Utopia");
```

Comment:
This is an anti-climatic name given the previous three puns!

Reply:
Fair point, I'll mull it over!